doc/source/ray-core/tasks/scheduling.rst

```diff
@@ -48,9 +65,29 @@ until the resource utilization is beyond a certain threshold and spread tasks af
         # Spread tasks across the cluster.
         [spread_function.remote() for i in range(100)]
 
+        @ray.remote
+        def node_function():
+            return 3
+
+        # Only run the task on the local node.
+        node_function.options(
+            scheduling_strategy=NodeSchedulingStrategy(
+                node_id = ray.get_runtime_context().node_id.hex(),
+                soft = False,
+            )
+        ).remote()
+
+        # Run the task on the remote node if possible.
+        node_function.options(
+            scheduling_strategy=NodeSchedulingStrategy(
+                node_id = remote_node_id,
+                soft = True,
+            )
+        ).remote()
```

Comment:
One more thing for this doc: can we link to the docs for APIs that return node IDs that the user can pass? Specifically the get_runtime_context call and the call to get all of the node IDs in the current cluster.

Reply:
Also, is this code runnable? I didn't see `remote_node_id` defined. We should make it runnable if possible!